index.html

```diff
@@ -0,0 +1,312 @@
+          lazy dog</p>
+      </div>
+      <div class="feature-card">
+        <div class="feature-card__head">
+          <img src="./assets/icons/bx_bx-stats.svg" alt="graph icon" class="feature-card__icon">
+          <h5 class="headline headline--dark headline--bolder">Digital marketing</h5>
+        </div>
+        <p class="description">the quick fox jumps over the lazy
+          dog the quick fox jumps over the
+          lazy dog</p>
+      </div>
+      <div class="feature-card">
+        <div class="feature-card__head">
+          <img src="./assets/icons/fa-solid_award.svg" alt="medal icon" class="feature-card__icon">
+          <h5 class="headline headline--dark headline--bolder">Digital marketing</h5>
+        </div>
+        <p class="description">the quick fox jumps over the lazy
+          dog the quick fox jumps over the
+          lazy dog</p>
+      </div>
+    </div>
+  </section>
+
+  <section class="services">
+    <div class="section-text">
+      <h1 class="section-text__heading">We are providing best
+        business service.</h1>
+      <p class="description">We know how large objects will act, but things
+        on a
+        small scale just do not act that way.</p>
+    </div>
+    <img class="services__image" src="./assets/images/card-item.png" alt="video image">
+    <div class="services__image-background">
+    </div>
+
+  </section>
+
+  <section class="people-section">
+    <div class="section-text">
+      <h1 class="section-text__heading">What Clients Say</h1>
+      <p class="description">Problems trying to resolve the conflict between
+        the two major realms of Classical physics: Newtonian mechanics</p>
+    </div>
+    <div class="people-section__card-container">
+      <div class="testimony-card">
+        <img src="./assets/icons/stars.svg" alt="rating stars" class="testimony-card__ratings">
+        <p class="description">Slate helps you see how many
+          more days you need to work to
+          reach your financial goal.</p>
+        <div class="testimony-card__footer">
+          <img src="./assets/images/Ellipse 2.png" alt="person" class="testimony-card__user-image">
+          <div class="testimony-card__user-details">
+            <a href="#" class="headline headline--colored">Regina Miles</a>
+            <h6 class="headline headline--dark">Designer</h6>
+          </div>
+        </div>
+      </div>
+      <div class="testimony-card">
+        <img src="./assets/icons/stars.svg" alt="rating stars" class="testimony-card__ratings">
+        <p class="description">Slate helps you see how many
+          more days you need to work to
+          reach your financial goal.</p>
+        <div class="testimony-card__footer">
+          <img src="./assets/images/Ellipse 2.png" alt="person" class="testimony-card__user-image">
+          <div class="testimony-card__user-details">
+            <a href="#" class="headline headline--yellow">Regina Miles</a>
```

Comment:
You forgot to change `headline--yelllow` to `headline--colored` like you did on line 139